### PR TITLE
Issues 295 + 296

### PR DESCRIPTION
**Pull request by AntoineVe**
Resolves issues 295 and 296.

### CODE REVIEW

**Comment by justinmayer** [comment]
The protocol-relative URL scheme is now considered to be an anti-pattern, since it's better to simply load the resource via HTTPS when possible. Ergo, please change this...

    //fonts.googleapis.com/css?family=Yanone+Kaffeesatz&subset=latin

... to:

    https://fonts.googleapis.com/css?family=Yanone+Kaffeesatz&subset=latin

... and then rebase+squash your commits. Thanks!

**Comment by AntoineVe** [comment]
Done it :-)

File: notmyidea-cms-fr/static/css/main.css [hunk 1]
```diff
@@ -10,12 +10,16 @@
 /* Imports */
 @import url("reset.css");
 @import url("pygment.css");
-@import url(http://fonts.googleapis.com/css?family=Yanone+Kaffeesatz&subset=latin);
+<<<<<<< HEAD
```
Comment (kylef) [inline notmyidea-cms-fr/static/css/main.css]: This includes a merge conflict.

**Comment by justinmayer** [comment]
Please [squash your commits](https://github.com/getpelican/pelican/wiki/Git-Tips), after which this pull request should be clear to merge.

**Comment by AntoineVe** [comment]
I'm totally lost in git... juste close this bug and I'will re-open if necessery a new issue...

**Comment by justinmayer** [comment]
Thanks for the fixes. Addressed via https://github.com/getpelican/pelican-themes/commit/a4ce53ca9bf12bbe292754218b2d9a5bdfd0f38c.

**Pull request closed without merging**